### ISSUE TITLE
Is the performance good or bad?

### ISSUE THREAD

**Issue by zhuqunxi** [junyanz/CycleGAN#85]
After training 140 epochs, 1000 images for each epoch, I got the image from horse to zebra. It seems not bad but also not good compared to the performance in the original paper (at least, the grass in the picture vanished). So, I am wondering in which processing may exist a bug or how can I improve its performance？I will appreciate it if you can help me. Thank you.
![image](https://user-images.githubusercontent.com/22273557/38780038-80f5539e-4103-11e8-84e2-451312cd8b33.png)
![image](https://user-images.githubusercontent.com/22273557/38780045-94db8b4e-4103-11e8-8ff2-a4fd955a06ca.png)
the following picture shows the error from the latest epochs. It seems the optimization is no longer forward.
![image](https://user-images.githubusercontent.com/22273557/38780115-820d538e-4104-11e8-9a04-0fd4600f094a.png)

**Comment by junyanz** [comment]
This looks reasonable. You have some checkboard artifacts. The artifacts will go away if you train the model for more iterations and a smaller learning rate.  You can also use upsampleConv to replace deConv as pointed out by this [post](https://distill.pub/2016/deconv-checkerboard/).

**Comment by zhuqunxi** [comment]
Great thanks for your help. I'll try it.
